app/code/Magento/Newsletter/Model/ResourceModel/Subscriber.php

```diff
@@ -157,6 +181,52 @@ public function loadByCustomerData(\Magento\Customer\Api\Data\CustomerInterface
         return [];
     }
 
+    /**
+     * Load subscriber by customer
+     *
+     * @param \Magento\Customer\Api\Data\CustomerInterface $customer
+     * @return array
+     */
+    public function loadByCustomerDataAndStore(\Magento\Customer\Api\Data\CustomerInterface $customer)
```

Comment:
Unfortunately this class marked as @api, so according to https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/ we can't new public methods into it